tests/tests/blocks.rs

```diff
@@ -423,4 +424,95 @@ mod full_block {
         assert_eq!(block.header.height.0, 1);
         assert_eq!(block.transactions.len(), 2 /* mint + our tx */);
     }
+
+    use fuel_core_types::{
+        fuel_asm::{
+            op,
+            RegId,
+        },
+        fuel_vm::SecretKey,
+    };
+
+    use rand::{
+        rngs::StdRng,
+        Rng,
+        SeedableRng,
+    };
+
+    #[tokio::test]
+    async fn too_many_transactions_are_split_in_blocks() {
```

Comment:
Can you divide this test into "Given", "When" "Then" sections? It makes it nicer to read.